test/form-builder/components/FormDetailContainer.spec.js

```diff
@@ -182,6 +182,18 @@ describe('FormDetailContainer', () => {
     expect(publishButton).to.have.length(0);
   });
 
+  it('should show preview button before get formData', () => {
```

Comment:
Should we write a test for preview button will appear after publish?

Reply:
yes added